### PR TITLE
Add  addtask and deletetask

### PR DESCRIPTION
**Pull request by LeopardMerkava**
Add `addtask` and `deletetask` feature

## Things to take note:

* Tests are not completed. This is to ensure that:
        1. Feature is put in first so work on `viewtask` can be started.
        2. Some behavior needs to be discussed. Adding test for those now will cause unnecessary repeat if behaviors are not agreed upon.

## Current added / changed / should be considered behavior:
Add:
- Can directly add multiple task
- Returns amount of task attached

Addtask:
- Can directly add multiple task

Deletetask:
- Can only delete 1 task (index would be wonky)

Edit:
- Cannot edit task (will throw an exception)

Task / TaskList:
- Only have string as name (for now)
- Is **NOT** considered in comparing people (Task list change a lot)

### CODE REVIEW

File: src/main/java/seedu/address/ui/PersonCard.java [hunk 1]
```diff
@@ -55,6 +58,7 @@ public PersonCard(Person person, int displayedIndex) {
         person.getTags().stream()
                 .sorted(Comparator.comparing(tag -> tag.tagName))
                 .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
+        taskList.setText(person.getTasks().toString());
```
Comment (limzk126) [inline src/main/java/seedu/address/ui/PersonCard.java:61]: I don't think this is necessary since I'm going to implement the side by side taskList window.

Comment (LeopardMerkava) [reply src/main/java/seedu/address/ui/PersonCard.java:61]: Aggreed! This is only a patch for me to see if my command works. Delete it once you implement the taskList!

**Comment by kflim** [comment]
LGTM, code seems fine

**Comment by LeopardMerkava** [comment]
Alright! Full acceptance, so I will merge it now.

**Pull request merged**